templates/transcript.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Template for transcripts.
+ * Intentionally barebones with the minimum html for use by tools.
+ *
+ * @package tenup_podcasting
+ */
+
+?>
+<!DOCTYPE html>
+<html lang="<?php echo esc_attr( get_locale() ); ?>">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>
+		<?php
+		printf(
+			/* translators: %s: The page title */
+			esc_html__( 'Transcript - %s', 'simple-podcasting' ),
+			get_the_title() // phpcs:ignore WordPress.Security.EscapeOutput 
```

Comment:
```suggestion
			wp_strip_all_tags( get_the_title() )
```

WP allows HTML in titles and browsers just render the tags as is, so this avoids tabs that look like this

<img width="360" alt="Screen Shot 2023-09-20 at 10 44 53 am" src="https://github.com/10up/simple-podcasting/assets/519727/3e1f5822-96a1-4e41-a57d-84097a0e53b2">

Reply:
Changed.